sokutei.h

```diff
@@ -0,0 +1,72 @@
+#ifndef SOKUTEI_BENCHMARK
+#define SOKUTEI_BENCHMARK
+
+#define MAX_COUNTERS 100
+#define COUNTER_LENGTH 32
+#define INT int
+
+#ifdef SOKUTEI_NO_STDLIB
+#ifndef SOKUTEI_STDLIB
+#define SOKUTEI_STDLIB
+
+int sokutei_strcmp(const char *a, const char *b){
+    int i = 0;
+    while(a[i] && b[i] && a[i] == b[i]) i++;
+    return a[i] - b[i];
+}
+#define STRCMP(a, b) sokutei_strcmp(a, b)
+
+char *sokutei_strcpy(char *a, const char *b){
+    int i;
+    for(i = 0; a[i] = b[i]; i++);
+    return a;
+}
+#define STRCPY(a, b) sokutei_strcpy(a, b)
+
+#endif //END OF SOKUTEI_STDLIB
+#else
+#include <string.h>
+#include <stdio.h>
+#define STRCMP(a, b) strcmp(a, b)
+#define STRCPY(a, b) strcpy(a, b)
+
+#endif //END OF SOKUTEI_NO_STDLIB
+
+char sokutei_int_counter_names[MAX_COUNTERS][COUNTER_LENGTH] = {'\0'};
+INT  sokutei_int_counters[MAX_COUNTERS] = {0};
+int  sokutei_int_number_of_counters = 0;
+
+void sokutei_add_counter(const char *ctr){
+    STRCPY(sokutei_int_counter_names[sokutei_int_number_of_counters++], ctr);
+}
+
+int sokutei_int_increment_counter(const char *ctr, int by){
```

Comment:
Invalid types. Both the return type and the type of the `by` parameter must be `INT`.